src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -159,14 +164,13 @@ const TokensMoveDialogForm = ({
     }
   }, [colonyAddress, tokenAddress, fromDomain, toDomain, loadTokenBalances]);
 
-  const [fromDomainTokenBalance, toDomainTokenBalance] = useMemo(() => {
+  const [fromDomainTokenBalance] = useMemo(() => {
     const token =
       tokenBalancesData &&
       tokenBalancesData.tokens.find(({ address }) => address === tokenAddress);
     const from = getBalanceFromToken(token, fromDomain);
-    const to = getBalanceFromToken(token, toDomain);
-    return [from, to];
-  }, [fromDomain, toDomain, tokenAddress, tokenBalancesData]);
+    return [from];
```

Comment:
if we returning just one value, should we just return from? and replace 
const [fromDomainTokenBalance]= with just 
const fromDomainTokenBalance=